components/constellation/constellation.rs

```diff
@@ -5388,29 +5428,26 @@ where
             })
     }
 
-    /// Re-send the frame tree to the compositor.
-    fn update_frame_tree_if_active(
+    /// Send the frame tree for the given browser to the compositor.
+    fn update_frame_tree_if_focused(
```

Comment:
This name is much clearer, but I think it also needs to reflect that it will cause the the TopLevelBrowsingContext to be focused if it isn't already.

Reply:
no longer needed as of 2afa67966ac13eb312cbf0fa616cbfe7c3790f4f